internal/throttler/throttler.go

```diff
@@ -16,6 +16,10 @@ type Throttler struct {
 	bytesPerSecond float64
 }
 
+func NewThrottlerBytesPerSecond(bytesPerSec float64) *Throttler {
+	return &Throttler{bytesPerSecond: bytesPerSec}
+}
+
 func NewThrottler(rate string) (*Throttler, error) {
```

Comment:
Do we need this func that takes a `string` now that we're parsing the bytes-per-sec at the "edge" of the CLI?

Reply:
No, so I've removed it 👍🏻